Food Receipe/Pages/index.html

```diff
@@ -1,13 +1,96 @@
-    <title>Document</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles.css" />
+    <script
+      src="https://kit.fontawesome.com/bb8e411a96.js"
+      crossorigin="anonymous"
+    ></script>
+    <script src="../index.js" defer></script>
+    <title>Recipe Finder</title>
+  </head>
+  <body>
+    <header>
+      <h1>Recipe Hub</h1>
+      <p>Search recipes by ingredients</p>
+    </header>
+    <main>
+      <section class="search">
+        <form action="" method="get">
+          <input
+            class="search__input"
+            type="text"
+            name="search"
+            placeholder="Enter an ingredient"
+          />
+          <div class="search-button">
+            <i class="fa-solid fa-magnifying-glass"></i>
+            <input type="button" value="Search" />
+          </div>
+        </form>
+      </section>
+      <section class="results">
+        <h2>
+          Your search results for
+          <span class="ingredient-searched"> "ingredient" </span>
+        </h2>
+        <div class="results__container">
+          <div class="results__result">
+            <img
+              src="https://picsum.photos/300/200"
+              alt="plate of delicious food"
+              class="result__image"
+            />
+            <h3 class="result__name">Meal Name</h3>
+            <button type="button" class="result__get-recipe">Get Recipe</button>
+          </div>
+          <div class="results__result">
+            <img
+              src="https://picsum.photos/300/200"
+              alt="plate of delicious food"
+              class="result__image"
+            />
+            <h3 class="result__name">Meal Name</h3>
+            <button type="button" class="result__get-recipe">Get Recipe</button>
+          </div>
+          <div class="results__result">
+            <img
+              src="https://picsum.photos/300/200"
+              alt="plate of delicious food"
+              class="result__image"
+            />
+            <h3 class="result__name">Meal Name</h3>
+            <button type="button" class="result__get-recipe">Get Recipe</button>
+          </div>
+        </div>
+        <dialog class="modal">
+          <p class="modal__close">&#10006;</p>
+          <h3 class="modal__name">Meal Name</h3>
+          <img
+            src="https://picsum.photos/300/200"
+            alt=""
+            class="modal__image"
+          />
+          <ul class="modal__category">
+            <li>category</li>
+            <li>category</li>
+          </ul>
+          <h4>Ingredients</h4>
+          <p class="modal__ingredients">Insert ingredients here.</p>
+          <h4>Instructions</h4>
+          <p class="modal__instructions">Insert instructions here.</p>
+          <a href="https://notion.so">Watch Video</a>
```

Comment:
Since we are going to perform these tasks dynamically after obtaining the results, we won't need the above code. can you remove it?

Reply:
The code is meant as a placeholder so when we work on the CSS we can preview how the style will look like. I will remove the placeholder code after I finish setting up the CSS.

Reply:
There is a reason why I made the container there. It is to give us the ability to style everything **before** you are done doing the API part. When you add the elements dynamically, please use the classes we already created, so that your elements will get styled as well. When you are ready to deploy your changes, THEN we can delete the whole result container. 
Basically we need the container to style what you create with the API. Does that make sense?